packages/core/src/client.rs

```diff
@@ -30,9 +27,4 @@ pub struct ClientConfig {
 
 pub trait Client: Invoker + UriResolverHandler {
   fn get_env_by_uri(&self, uri: &Uri) -> Option<&Env>;
-  fn load_wrapper(
```

Comment:
I think load_wrapper is a useful method of the Client trait.
The client can:
- load wrappers
- invoke wrappers
- invoke uris